universe/assets_3d/assets_3d.go

```diff
@@ -157,6 +157,23 @@ func (a *Assets3d) RemoveAssets3d(assets3d []universe.Asset3d, updateDB bool) er
 	return nil
 }
 
+func (a *Assets3d) RemoveAssets3dByIDs(ids []uuid.UUID, updateDB bool) error {
```

Comment:
if it's useful method, then change `ids` name properly and add it to interface, or ask me and I'll create one by myself